charts/fluentbit-operator/templates/fluentbit-operator-deployment.yaml

```diff
@@ -30,67 +30,55 @@ spec:
         - /bin/sh
         - '-c'
         - set -ex;
-          echo CONTAINER_ROOT_DIR=$(docker info -f '{{`{{.DockerRootDir}}`}}') > /fluentbit-operator/fluent-bit.env
+          echo CONTAINER_ROOT_DIR=$(docker info -f '{{`{{.DockerRootDir}}`}}') > /fluent-operator/fluent-bit.env
         volumeMounts:
         - name: env
-          mountPath: /fluentbit-operator
+          mountPath: /fluent-operator
         - name: dockersock
           readOnly: true
           mountPath: /var/run/docker.sock
       {{- else if eq .Values.containerRuntime "containerd" }}
       volumes:
       - name: env
         emptyDir: {}
-      - name: containerdsock
-        hostPath:
-          path: /var/run/containerd/containerd.sock
-      - name: crictl
-        hostPath:
-          path: {{ .Values.logPath.crictlPath }}
       initContainers:
       - name: setenv
         image: {{ .Values.operator.initcontainer.image }}:{{ .Values.operator.initcontainer.tag }}
         command:
         - /bin/sh
         - '-c'
         - set -ex;
-          echo CONTAINER_ROOT_DIR=$(crictl info --output go-template --template '{{`{{.config.containerdRootDir}}`}}') > /fluentbit-operator/fluent-bit.env
+          echo CONTAINER_ROOT_DIR={{ .Values.logPath.containerd }} > /fluent-operator/fluent-bit.env
```

Comment:
Refer to the changes in latest PR https://github.com/fluent/fluentbit-operator/pull/202/commits/ac62edccaa64e20c806bee9b9eb247261ca36a6f#diff-11a3a526accdc2c72ddd4b8cf745ad478e23fa61a1ad80dd62d532e47cae40a7R33

Reply:
fixed. This is used because it is the field that is the `containerd` or `crio` field. For `docker` environments, this field has been modified.